network-api/networkapi/mozfest/templates/fragments/blocks/mozfest_tickets_block.html

```diff
@@ -0,0 +1,31 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+    <div class="tw-bg-black tw-dark tw-py-16">
+        <div class="tw-container">
+            <h2 class="tw-mb-14">{{ title }}</h2>
+            <div class="tw-row">
+                {% for tier in self.tickets %}
+                    {% with count=self.tickets|length %}
+                        <div class="tw-mb-12 tw-w-full{% if count > 2 %} large:tw-w-1/3{% else %} large:tw-w-1/2{% endif %} tw-px-8{% if tier.sticker_text %} tw-relative{% endif %}">
+                            <div class="tw-bg-festival-black-100 tw-p-8 tw-h-full tw-flex tw-flex-col tw-items-start">
+                                <h3 class="tw-h2-heading">{{ tier.price }}</h3>
+                                <p class="tw-mt-0">{{ tier.group }}</p>
+                                <div class="[&>*]:tw-text-sm">
```

Comment:
Interesting. Does the font size not cascade down? Or it there another rule at place we have to override?

Reply:
The text that appears here will come from a rich text input which means I can't apply classes directly to any of the elements that get output - `p` or `li` tags for example. The default font-size set on these elements is too large compared to the design hence why the font size needs to be set higher up on this element. You have made me realise that the use of the child combinator is wrong due to the `rich-text` div that will be output.

Reply:
Updated in https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11537/commits/e3ac3cf1985370e01bf04fa3af78a083e3c2863a